### PR TITLE
Added form id property to fix getFormGroup

### LINKED ISSUE #86: [fix]: demo app submit handling
Right now when you click Submit on the demo app, you'll get a message:

```
Hi, My username is undefined. This Library is not awesome.
```

Ahhhhhhh 😱

Well, don't panic. The fix is simple... you just need to modify the `getFormGroup` method and the `Form.astro` component

_✨ Hint: `getFormGroup` is now returning `undefined` because it is not finding an element with the passed name._

### PR DESCRIPTION
**Pull request by itsagift**
type(scope): description

Fixes #86 

Description of changes: 
- Might be a makeshift fix -- I simply added a way to add id with the form name to the "form" element created by Form.astro if it is not an array -- if FormGroup is an array, each individual form is a fieldset and has an id with its name, and therefore can be targeted by getFormGroup.

Tag a reviewer: @ayoayco 

Tasks:
- [X] I have ran the tests to make sure nothing is broken (see CONTRIBUTING.md)
- [X] I have ran the the linter to make sure code is clean (see CONTRIBUTING.md)

### CODE REVIEW

File: packages/astro-reactive-form/Form.astro [hunk 1]
```diff
@@ -12,9 +12,10 @@ export interface Props {
 const { submitControl, formGroups: form, theme } = Astro.props;
 
 const formTheme = theme ?? 'light';
+const formName = !Array.isArray(form) ? form?.name : null;
 ---
 
-<form class={formTheme}>
+<form id={formName} class={formTheme}>
```
Comment (ayoayco) [inline packages/astro-reactive-form/Form.astro:18]:
```suggestion
<form id={formName} name={formName} class={formTheme}>
```

**Review by ayoayco — CHANGES_REQUESTED** [review]
Hi @itsagift! I was actually thinking this same thing. I have one suggestion to add a `name` as well. :)

**Comment by ayoayco** [comment]
Hi @itsagift! There were significant developments today that makes this PR and issue obsolete, so I have to close this now. Other issues are still open. Let me know if I can help find an interesting contribution for you. :)

**Comment by itsagift** [comment]
@ayoayco No problem!

**Pull request closed without merging**